rocAL/rocAL_pybind/amd/rocal/plugin/pytorch.py

```diff
@@ -92,7 +94,7 @@ def __init__(self, pipeline, tensor_layout = types.NCHW, reverse_channels = Fals
                     self.out = torch.empty((self.bs*self.n, int(self.h/self.bs), self.w, self.p), dtype=torch.float32)
                 elif self.tensor_dtype == types.FLOAT16:
                     self.out = torch.empty((self.bs*self.n, int(self.h/self.bs), self.w, self.p), dtype=torch.float16)
-                self.labels = torch.empty(self.labels_size, dtype = torch.int32, device = torch_gpu_device)
+                self.labels = torch.empty(self.labels_size, dtype = torch.int32)
```

Comment:
why is this changed back to cpu allocation. I thought only TF needed this change

Reply:
This was a bug Rajy.

I had used gpu device for a cpu tensor allocation.

We do have the gpu allocation also.